pac4j-saml/src/main/java/org/pac4j/saml/metadata/SAML2IdentityProviderMetadataResolver.java

```diff
@@ -71,27 +75,31 @@ public final MetadataResolver resolve() {
                     path = "/" + path;
                 }
                 resource = ResourceHelper.of(new ClassPathResource(path));
-            } else if (this.idpMetadataPath.startsWith("file:")) {
-                resource = ResourceHelper.of(new FileSystemResource(this.idpMetadataPath));
-            } else if (this.idpMetadataPath.startsWith("http")) {
+            }  else if (this.idpMetadataPath.startsWith("https")) {
```

Comment:
You're right: HTTPS is the foundation of security. But what about the development environment and local tests? I guess it would be better for this kind of scenario to allow `http`:  don't you think?